src/cmd/ksh93/sh/io.c

```diff
@@ -1881,6 +1881,24 @@ void sh_vexsave(Shell_t *shp,int fn,int fd,Spawnvex_f vexfun, void *arg)
 	}
 }
 
+// Return the lowest numbered fd that is equal to or greater than the requested
```

Comment:
Rest of the ksh code uses ```/*...*/``` type of comments.

Reply:
Yep, but not for much longer :smile: